docs/spec/channel/version_060.md

```diff
@@ -0,0 +1,137 @@
+# Channel Spec (Version 0.6.0)
+
+## Background
+
+The current system consists of two major APIs, the `Channel` and their `ClusterChannelProvisioner`. Both are living in the `eventing.knative.dev/v1alpha1` API Group.
+
+### The Channel
+
+A `kind: Channel` logically receives events on its input domain and forwards them to its subscribers. Below is a specification for the different parts of the `kind: Channel`.
+
+### The ClusterChannelProvisioner
+
+Describes an abstract configuration of a Source system which produces events or a Channel system that receives and delivers events.
```

Comment:
Should "Source" be "Importer" or is that change still pending?
Are we folding configuration of source and channel together? If so it seems like it might be time to identify some common "endpoint" configuration type that describes either the client or server end of some protocol connection, where the direction of event flow is *not* assumed to be client->server (it will be for HTTP, but not for other protocols)

Reply:
that's the old spec ... matching old terms ?